### DISCUSSION TITLE
Web postal is always inaccessible

### DISCUSSION THREAD

**Discussion by Infouzi** [postalserver/postal#1755] [Help with using Postal]
Hi everyone,

I just tried and went through the installation process. Everything seems to be fine, except that when i try to access the postal_web container.. i can't.

### postal_web_1  container logs:
<img width="506" alt="Capture d’écran 2021-12-05 à 11 22 48 AM" src="https://user-images.githubusercontent.com/61884413/144742645-c4c6dfbe-af83-40bb-9805-4010826244f1.png">

Any help would be welcome, thanks.

**Comment by willpower232** [comment]
Have you set up caddy or nginx to point at Postal? What is the output of `postal status`?

**Reply by Infouzi** [reply]
yes i have. i tried with Caddy and Nginx Proxy Manager too.. but nothing happened.

### This is the `postal status` output:
<img width="495" alt="Capture d’écran 2021-12-06 à 3 22 09 PM" src="https://user-images.githubusercontent.com/61884413/144862874-cbb44e73-5577-4c07-8578-9dd277473e66.png">

**Reply by willpower232** [reply]
what do you mean by nothing happened? did you get a browser error? what was your caddy configuration?

**Reply by Infouzi** [reply]
When i say nothing happened, i meant that when i go to to browser and type my domain name, it doesn't work.

Here  is my Caddyfile config:

mail.mydomain.com {
  reverse_proxy 127.0.0.1:5000
}

**Reply by willpower232** [reply]
That Caddyfile looks fine. When it doesn't work in your browser, what error message did your browser have?

**Reply by Infouzi** [reply]
We can’t connect to the server at `mail.mydomain.com`.

There is no http adequate response (no 1xx,2xx,3xx,4xx,5xx response).
